.github/mergify.yml

```diff
@@ -20,6 +21,7 @@ pull_request_rules:
 
   - name: Ask to resolve CI failures
     conditions:
+      - -closed
       - or:
           - check-failure=test-suite-success
           - check-skipped=test-suite-success
```

Comment:
Maybe we need to mirror the opposite of this condition in the ready-for-review task?

Reply:
Added